app/shared/resource-card/ResourceCard.js

```diff
@@ -49,12 +49,10 @@ class ResourceCard extends Component {
   };
 
   handleLinkClick = () => {
-    const scrollTop = window.pageYOffset
-      || document.documentElement.scrollTop
-      || document.body.scrollTop;
+    window.scrollTo(0, 0);
     const { location, history } = this.props;
     const { pathname, search } = location;
-    history.replace({ pathname, search, state: { scrollTop } });
```

Comment:
Have you checked if this `scrollTop` method is used anywhere else?

Reply:
It is used in SearchPage. I have now keep scrollTop as it is before and used ```window.scrollTo(0,0)``` to take the browser view to the top.